### PR TITLE
Consistent license across all the project

### PR DESCRIPTION
**Pull request by abdonrd**
### Summary

Consistent license across all the project.

### Details and comments

PR requested by @jaygambetta.
~Don't merge yet: there are still some changes that he will make.~

### CODE REVIEW

**Comment by jaygambetta** [comment]
i could not see changes to the contribution file. We need to have something in there that we are using a CLA

**Comment by jaygambetta** [comment]
Contributor License Agreements

We'd love to accept your code! Before we can, we have to get a few legal requirements sorted out.

Before we can accept a pull request from you, you'll need to sign a Contributor License Agreement (CLA). It is an automated process and you only need to do it once.

NOTE:  If you work for a company that wants to allow you to contribute your work, then you'll need to sign a corporate CLA (link to download file) and email it to us at qiskit@us.ibm.com.

**Comment by abdonrd** [comment]
Delegate the CLA to #552.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,7 @@
-# Copyright 2017 IBM RESEARCH. All Rights Reserved.
+# Copyright (c) 2017, IBM.
```
[the remark this answers is not in the document]
Comment (abdonrd) [reply .travis.yml]: It's already done! 🎉

**Comment by diego-plan9** [comment]
Wonderful, thanks @abdonrd and @jaygambetta !

**Pull request merged**